README.md

```diff
@@ -19,7 +19,7 @@ npm install humanize-duration
 With `require` (like in Node or with common build systems):
 
 ```js
-const humanizeDuration = require("humanize-duration");
+const humanizeDuration = require('humanize-duration');
```

Comment:
I don't think the readme should've changed as part of this pull request, except for adding your name to the credits. Could you revert?

Reply:
ok. I'm on it